lib/usdUfe/utils/usdUtils.cpp

```diff
@@ -107,6 +107,36 @@ void replaceInternalReferencePath(
     }
 }
 
+void removeInternalReferencePath(
+    const UsdPrim&            deletedPrim,
+    const SdfReferencesProxy& referencesList,
+    SdfListOpType             op)
+{
+    // set the listProxy based on the SdfListOpType
```

Comment:
In this file, please always compare with the function just above the new one to see the difference between cleaning up and renaming.